src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_INT = 2147483647;
+    private Random random = new Random();
+    private Figure[] figures = { new Circle(random.nextInt(MAX_INT)),
+            new IsoscelesTrapezoid(random.nextInt(MAX_INT),
+                    random.nextInt(MAX_INT),
+                    random.nextInt(MAX_INT)),
+            new Rectangle(random.nextInt(MAX_INT),
+                    random.nextInt(MAX_INT)),
+            new RightRectangle(random.nextInt(MAX_INT),
+                    random.nextInt(MAX_INT)),
+            new Square(random.nextInt(MAX_INT)) };
```

Comment:
remove this variable, you should generate random figures directly in getRandomFigure() dynamically, you don't need to return the same figures, based on the random index you should return a random figure